AssetInformationApi/V1/Controllers/AssetInformationApiController.cs

```diff
@@ -92,5 +97,24 @@ public async Task<IActionResult> AddAsset([FromBody] AddAssetRequest asset)
 
             return StatusCode(StatusCodes.Status201Created, result);
         }
+
+        [ProducesResponseType(StatusCodes.Status204NoContent)]
+        [ProducesResponseType(StatusCodes.Status400BadRequest)]
+        [ProducesResponseType(StatusCodes.Status404NotFound)]
+        [ProducesResponseType(StatusCodes.Status500InternalServerError)]
+        [HttpPatch]
+        [Route("{id}")]
+        [LogCall(LogLevel.Information)]
+        public async Task<IActionResult> PatchAsset([FromRoute] EditAssetByIdRequest query, [FromBody] AssetDb asset)
```

Comment:
Can you please create a separate update asset request class (as part of the asset-shared) and use that instead of the AssetDb model class? The request class should only include fields that should be editable.